ArduCopter/control_auto.cpp

```diff
@@ -580,24 +580,20 @@ uint8_t Copter::get_default_auto_yaw_mode(bool rtl)
 
         case WP_YAW_BEHAVIOR_NONE:
             return AUTO_YAW_HOLD;
-            break;
 
         case WP_YAW_BEHAVIOR_LOOK_AT_NEXT_WP_EXCEPT_RTL:
             if (rtl) {
                 return AUTO_YAW_HOLD;
             }else{
```

Comment:
else after a return makes no sense neither. And it would be clearer all branches are handled. Feel free to send a follow up to this patch if you want.